### DISCUSSION TITLE
Is It Possible to Have no Column Headers?

### DISCUSSION THREAD

**Discussion by slimandslam** [sajrashid/hooks#93] [Q&A] [answered]
My data has no column headers. By default, it appears that you insert  numbers [0,1,2,3...]  as the column headers. Can I get rid of those somehow?

**Comment by sajrashid** [comment]
Hi @slimandslam, not too sure if i understand, can you show me what you json looks like, or do you have a codesandbox ?

**Comment by sajrashid** [comment, marked as answer]
@slimandslam 

 theadCss: "hideHeader",

[SandBox](https://codesandbox.io/s/hide-header-tailwind-n3y5t?file=/index.js:1130-1154)
